irony.el

```diff
@@ -99,6 +99,19 @@ automatically buffer-local wherever it is set."
   "C/C++ minor mode powered by libclang."
   :group 'c)
 
+(defcustom irony-allow-inaccessibles nil
```

Comment:
Also, I take it you want to record the availability  when the candidate is anything other than not-available since in Irony.cpp we `continue' the loop to save unnecessary processing?